### PR TITLE
Remove hydrator and update commands.

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ==========================================================
-Creating Hydrator Applications using CDAP System Artifacts
+Creating Pipeline Applications using CDAP System Artifacts
```
Comment (bdmogal) [inline README.rst]: Creating CDAP Pipelines using...

File: README.rst [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 ==========================================================
 
 Using the built-in ``cdap-data-pipeline`` and ``cdap-data-streams`` system artifacts, you can
-create Cask Hydrator Pipelines (Hydrator Applications) with just a JSON configuration file. CDAP ships
+create CDAP Pipelines (CDAP Applications) with just a JSON configuration file. CDAP ships
```
Comment (bdmogal) [inline README.rst]: I don't think the text in parantheses is necessary

File: README.rst [hunk 3]
```diff
@@ -65,8 +65,8 @@ What You Will Need
 
 Let's Begin!
 ============
-For these guides, we will use the CDAP CLI to create and manage Hydrator Applications. The CLI
-commands assume that the ``cdap-cli.sh`` script is available on your PATH. If this is not
+For these guides, we will use the CDAP CLI to create and manage CDAP Pipeline Applications. The CLI
```
Comment (bdmogal) [inline README.rst]: remove applications

File: README.rst [hunk 4]
```diff
@@ -75,11 +75,11 @@ or, from within the <CDAP home> directory::
...
   $ export PATH=${PATH}:`pwd`/bin
 
-If you haven't already started a standalone CDAP installation, start it with the command::
+If you haven't already started a Standalone CDAP installation, start it with the command::
 
-  $ cdap.sh start
+  $ cdap sdk start
 
-Now navigate to the Hydrator Application that you want to create and you will find further
+Now navigate to the CDAP Application that you want to create and you will find further
```
Comment (bdmogal) [inline README.rst]: CDAP Pipeline?

File: RealtimeKafkaToTPFSAvro/README.rst [hunk 5]
```diff
@@ -53,12 +53,12 @@ Starting CDAP
...
-  $ cdap.sh start
+  $ cdap sdk start
   
 
-Creating a Hydrator Application using the CDAP CLI
-==================================================
-From within the CDAP CLI, create a Hydrator application named ``flightIngest``, replacing
+Creating a CDAP Application using the CDAP CLI
+==============================================
+From within the CDAP CLI, create a pipeline application named ``flightIngest``, replacing
```
Comment (bdmogal) [inline RealtimeKafkaToTPFSAvro/README.rst]: remove application

File: RealtimeKafkaToTPFSAvro/README.rst [hunk 6]
```diff
@@ -53,12 +53,12 @@ Starting CDAP
...
 If you haven't already, start your CDAP installation::
 
-  $ cdap.sh start
+  $ cdap sdk start
   
 
-Creating a Hydrator Application using the CDAP CLI
-==================================================
-From within the CDAP CLI, create a Hydrator application named ``flightIngest``, replacing
+Creating a CDAP Application using the CDAP CLI
```
Comment (bdmogal) [inline RealtimeKafkaToTPFSAvro/README.rst]: create a CDAP Pipeline...

File: RealtimeTwitterToHBase/README.rst [hunk 7]
```diff
@@ -55,7 +55,7 @@ Twitter source:
 
 Creating and Running
 ====================
-Create a Hydrator application named ``tweetApp`` (replace <version> with your CDAP version) using the CDAP CLI::
+Create a pipeline application named ``tweetApp`` (replace <version> with your CDAP version) using the CDAP CLI::
```
Comment (bdmogal) [inline RealtimeTwitterToHBase/README.rst]: remove application

File: README.rst (hunk 1 above)
Comment (awholegunch) [reply README.rst]: Fixed.

File: README.rst (hunk 2 above)
Comment (awholegunch) [reply README.rst]: Fixed.

File: README.rst (hunk 3 above)
Comment (awholegunch) [reply README.rst]: fixed.

File: README.rst (hunk 4 above)
Comment (awholegunch) [reply README.rst]: fixed.

File: RealtimeKafkaToTPFSAvro/README.rst (hunk 5 above)
Comment (awholegunch) [reply RealtimeKafkaToTPFSAvro/README.rst]: fixed.

File: RealtimeKafkaToTPFSAvro/README.rst (hunk 6 above)
Comment (awholegunch) [reply RealtimeKafkaToTPFSAvro/README.rst]: Fixed.

File: RealtimeTwitterToHBase/README.rst (hunk 7 above)
Comment (awholegunch) [reply RealtimeTwitterToHBase/README.rst]: fixed.

**Review by bdmogal — APPROVED** [review]

**Pull request merged**